Makefile

```diff
@@ -9,13 +9,19 @@ VERSION := $(shell echo ${VERSION} | sed -e "s/^v//")
 ifeq ($(VERSION),)
 VERSION := `cat CWAGENT_VERSION`
 endif
+
+# Determine agent build mode, default to PIE mode
+ifndef CWAGENT_BUILD_MODE
```

Comment:
I think maybe we should default to non pie build mode, also can you update `Dockerfile` like https://github.com/aws/amazon-cloudwatch-agent/compare/master...pingleig:k8s-skip-metric?expand=1 to see if build from source using docker works (btw: you might need to switch network to solve go mod getting blocked problem)

Reply:
Updated the default build mode to `default` instead of `pie`